src/setup/templates/rgw.logrotate.tmpl

```diff
@@ -0,0 +1,13 @@
+TMP_LOG_PATH/*.log
+{
+    rotate 7
+    maxsize 50M
+    weekly
+    compress
+    missingok
+    copytruncate
+    postrotate
+        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
+    endscript
+    su root ceph
+}
```

Comment:
@cdeshmukh @sachitanands I have configured the size as 50M for now,
Let me know if any configurational change is required.

Reply:
50M for one log file is too large i feel, 10MB file is what we had been following as safer size, and maintain more number of rotated files to maintain to cover similar amount of logs eventually

Reply:
changed the size to 10M